django_prometheus/exports.py

```diff
@@ -88,7 +90,10 @@ def SetupPrometheusEndpointOnPortRange(port_range, addr=''):
         thread.daemon = True
         thread.start()
         logger.info('Exporting Prometheus /metrics/ on port %s' % port)
-        return  # Stop trying ports at this point
+        return port  # Stop trying ports at this point
+    logger.info('Cannot export Prometheus /metrics/ - '
```

Comment:
@seansawyer  This feels like it should be a warning, wdyt ?

Reply:
@asherf I had it as a warning initially, then lowered to info...didn't want to be too alarmist. 😆 But since you agree, I will bump it back to warning.

Reply:
Done